opa-manager-operator/files/constraintpodstatuses.status.gatekeeper.sh.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Seems like both charms are defining the same CRDs. Maybe remove the CRDs from one of them?

Reply:
Please remove duplicate CRDs.

Reply:
removed on audit